content/applications/finance/payment_providers/razorpay.rst

```diff
@@ -23,6 +23,11 @@ Configuration on Razorpay Dashboard
    checkboxes are ticked.
 #. Click on :guilabel:`Create Webhook` to finalize the configuration.
 
+.. note::
+   The :guilabel:`Recurring payments` feature must be activated if you want to make recurring payments.
```

Comment:
Add a [custom anchor](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#custom-anchors) to reference this admonition from the provider's view.